### ISSUE TITLE
Exemplar Fine-Tuning for 3D Human Model Fitting Towards In-the-Wild 3D Human Pose Estimation

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#18]
1.どんな論文? /What does this work do?

- https://arxiv.org/abs/2004.03686
- 3DV 2021
- Introduce Exemplar Fine-Tuning (EFT), which combines the re-projection accuracy of fitting methods like SMPLify with a 3D pose prior implicitly captured by a pre-trained 3D pose regressor network.

2.先行研究と比べてどこがすごい? /What makes this work greater than existing works?

![0c7fc92db5b5183bc18ee9d67cbff13](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/ce1e44a1-258a-4d68-8a86-fe5649eb1ecf)

- Providing large scale and high quality pseudo-GT 3D pose annotations that  are sufficient to train state-of-the-art regressors without in door 3D datasets; 
- Demonstrate that the COCO with pseudo annotations generated by EFT is sufficient to train state-of-the-art 3D pose regressor networks by itself, outperforming previous methods trained on the combination of datasets with 3D and 2D groundtruth on challenging benchmarks such as 3DPW.

3.技術や手法のキモはどこ? /What is the heart of this technology or method?

Exemplar Fine-Tuning combines the advantages of fitting and regression methods. The idea is to interpret the network **Φ** as a re-parameterization of the model as a function of the network parameters **w**. With this, as where:
![33b867cb03d011a46dfefaf2c6fc68d](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/9854b9c1-8b11-40ee-88f2-ed76f8f9d9e1)

4.どうやって有効だと検証した? /How is this work validated?

In table 5, carry out a quantitative evaluation on the 3DPW dataset. For initialization, use various regressors pre-trained on a number of different datasets, providing different initialization states. Then use EFT and SMPLify post-processing to fit the same set of 2D keypoint annotations, obtained automatically by means of the OpenPose detector.
![421c9fd7c3d803e13f41895edf87e1b](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/9a77db2d-0610-43a9-9618-714a92c7dde8)
Specifically, if the initialization is already good, as in the last row of table 5, SMPLify degrades the performance, particularly when the pose prior is used. This illustrates the difficulty in balancing data and prior terms, which may be difficult to do in practice. In contrast, EFT does not suffer from such issues, improving the accuracy in all cases, regardless of the quality of the initialization. This is also true when the shape regularizer is removed entirely.

As in the previous experiment(table 5), use various regressors to provide different initialization states, then use EFT and SMPLify for post-processing. The results are shown in table 6, showing analogous conclusions to the result of main paper
![21c50d56fd8b90ac127307f492f3960](https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/a74ffa09-f80e-462d-bdf4-98f47e0c4322)

5.議論はある? /Any discussion for this work?

6.読んでいてわからなかったところは? /What don't you understand for this paper?

まだ十分に理解できていないので、後で読み続ける必要があります

7.公開コードやデータセットは? /Public codes or datasets?

https://github.com/facebookresearch/eft

8.次に読むべき論文は? /What should we read next
